mutation-ready.ts

```diff
@@ -0,0 +1,68 @@
+declare global {
+  interface Window {
+    WebKitMutationObserver: MutationObserver;
+  }
+}
+
+type MutationElementReady = Element & {
+  ready: boolean;
+};
+
+interface Listener {
+  selector: string;
+  callback: (element: Element) => void;
+}
+
+const listeners: Listener[] = [];
+const document: Document = window.document;
+const MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
+let observer: MutationObserver | null;
```

Comment:
This is never actually `null`. It starts as `undefined` and then gets assigned an observer object. `null` should be removed from the type signature.